.azure-pipelines/stage/prechecks.yml

```diff
@@ -82,6 +82,9 @@ jobs:
       - template: ../gpg.yml
         parameters:
           condition: and(not(canceled()), eq(variables['CI_TARGET'], 'docs'))
+          # Reuse key downloaded above
+          downloadKey: false
+          name: MaintainerGPGKey2
```

Comment:
Is this necessary? if the first time the download task is run and at second time the task is not run, I am not sure what will be the value of `<task-name>.outputSecureFile`.

Reply:
yes this is necessary

without it, even if its not run, a 2nd job of the same name is created and azp dies

Reply:
Then would be better to document it.

fragile azp...they never think the users may run same task multiple times?

Reply:
beyond ...

>           # Reuse key downloaded above

?

i would prefer to find a way that we can just set `downloadKey: false` and not have to set the name - ie the name is then random or some such - but i didnt figure a way to do that so this is the workaround

Reply:
thinking about this further ...

as `downloadKey` is "early-bound" we can almost certainly use it as a condition to disable the job entirely

it means that this can never be late-set (eg cant be used in a matrix or as a result of any template expression)

as this will require a bit of testing i would prefer to do it in a follow up